app/models/requests/page.rb

```diff
@@ -17,8 +17,7 @@ def requestable_by_all?
   end
 
   def item_commentable?
-    return super unless origin == 'SAL-NEWARK'
-    !barcoded_holdings?
+    SULRequests::Application.config.item_commentable_libraries.include?(origin) || super
```

Comment:
I wonder if this should actually be at the request class level (also potentially some other of these library specific behaviors).  The reason is, if a SAL-NEWARK item were to get a hold/recall for some reason I **think** we would want this behavior to apply there as well.

cc: @jvine in case she has any insight (and about other things that are potentially consistent across all request types based on library level attributes).